sample/CsmpAgentLib_sample_util.c

```diff
@@ -0,0 +1,670 @@
+#include <stdio.h>
```

Comment:
Any specific reason for moving these from sample/CsmpAgentLib_sample.c to this new file? 
Original thought was to keep everything related to the sample app in one CsmpAgentLib_sample.c file. This file would not only have funcs handling the CSMP service handling but also the GET/PUT implementations as well.
What are your thoughts on this?

Reply:
Hi! I thought we should keep CsmpAgentLib_sample.c only for the thread functions, since there will be more due to further target (efr32). Without separating the things the sample app c file will include  more thousands lines.
An other reason is the abstraction, what will be necessary later for efr32 target to customise the TLV handlers. It has been implemented in internal repo already to abstract the IPv6 address TLV for example.

Reply:
True that TLV GET/PUT functions will increase over time as we keep implementing new TLV support. If we are expecting more code to handle the sample app's infra, we could go with the current approach - so we have all TLV GET/PUT implementations in a separate file.